dev/integration_tests/flutter_gallery/lib/demo/material/reorderable_list_demo.dart

```diff
@@ -150,8 +150,7 @@ class _ListDemoState extends State<ReorderableListDemo> {
           key: Key(item.value),
           height: 100.0,
           width: 100.0,
-          child: CircleAvatar(child: Text(item.value),
-            backgroundColor: Colors.green,
+          child: CircleAvatar(backgroundColor: Colors.green,child: Text(item.value),
```

Comment:
formatting: either put each property on an individual line or remove the trailing `,` and move the `),` up one line.